auth/common.cc

```diff
@@ -64,7 +64,7 @@ static future<> create_metadata_table_if_missing_impl(
     static auto ignore_existing = [] (seastar::noncopyable_function<future<>()> func) {
         return futurize_invoke(std::move(func)).handle_exception_type([] (exceptions::already_exists_exception& ignored) { });
     };
-    auto& db = qp.db();
+    auto db = qp.db();
```

Comment:
The change in this source file is a bit underwhelming... Wasn't the intention that we can get rid of `#include <database.hh>`? 
Or maybe this will just come later and you started with just the simplest transformations in each source file?

Reply:
I focused on cql3/. I'll tackle other coordinator-side modules later (thrift, transport, redis)